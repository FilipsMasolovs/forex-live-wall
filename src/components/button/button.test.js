import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Button from './button';

it('renders without crashing', () => {
  const parent = document.createElement('div')
  ReactDOM.render(<Button buttonText='Search' />, parent)
})

it('renders correctly', () => {
  const tree = renderer
    .create(<Button buttonText='Search' />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})