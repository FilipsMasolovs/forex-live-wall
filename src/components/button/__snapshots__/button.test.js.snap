// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="forex-live-wall--button"
>
  Search
</button>
`;
